strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/GroupRepositoryProvider.java

```diff
@@ -386,4 +386,13 @@ public Long count(RepositorySearchRequest searchRequest)
         return artifactEntryService.countCoordinates(storageRepositoryPairSet, searchRequest.getCoordinates(),
                                                      searchRequest.isStrict());
     }
+
+    @Override
+    public Path getPath(String storageId,
+                        String repositoryId,
+                        String artifactPath)
+           throws IOException
+    {
+        return null;
```

Comment:
`return null;`

Please `throw new UnsupportedOperationException()` instead of returning `null`

Reply:
+1 : For this and all other occurrences.